velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -900,6 +900,13 @@ TEST_F(StringTest, translate) {
   testTranslate({"abc", std::nullopt, "\u2029"}, std::nullopt);
   testTranslate({"abc", "\u2028", std::nullopt}, std::nullopt);
   testTranslate({std::nullopt, "\u2028", "\u2029"}, std::nullopt);
+  // Test corner case: there is chance the input contains invalid utf8 chars,
+  // which is generate by other functions like url_decode. We need to make sure
+  // they would not cause wrong results or failure. Below is an example.
```

Comment:
Perhaps document the behavior for invalid UTF-8 as the mask function.

https://github.com/facebookincubator/velox/blob/205cbdf0792f92f6468a5d51b00be63bb39f42b7/velox/docs/functions/spark/string.rst?plain=1#L155

It should be noted that we find Spark does not define the handling of invalid UTF-8, so below documentation has been added in Velox. cc: @gaoyangxiaozhu 

https://github.com/facebookincubator/velox/blob/205cbdf0792f92f6468a5d51b00be63bb39f42b7/velox/docs/functions/spark/string.rst?plain=1#L9-L10